main.py

```diff
@@ -210,6 +210,10 @@ async def ws_type_command(db, websocket, data):
             sw.on()
         else:
             sw.off()
+    if thing.type in ["shellytrv"]:
+        valve = db.query(Thing).get(thing_id)
```

Comment:
(This is also a bug in the `if` block above.) We don't need to fetch the `Thing` again as we already fetched it in [Line 202](https://github.com/iot-home-control/system-backend/pull/1/commits/19a46284d9aa03c0b046d897e76c83757705d5b2#L202).